e2e/test/scenarios/dashboard/dashboard-reproductions.cy.spec.js

```diff
@@ -1500,3 +1504,46 @@ describe("issue 42165", () => {
     });
   });
 });
+
+describe("issue 47170", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.request("POST", `/api/bookmark/dashboard/${ORDERS_DASHBOARD_ID}`);
+
+    createDashboard({ name: "Dashboard A" }, { wrapId: true }).then(
+      dashboardId => {
+        cy.request("POST", `/api/bookmark/dashboard/${dashboardId}`);
+      },
+    );
+
+    cy.intercept(
+      {
+        method: "GET",
+        url: "/api/dashboard/*",
+        middleware: true,
+      },
+      req => {
+        req.on("response", res => {
+          res.setThrottle(2);
+        });
+      },
+    );
+  });
+
+  it("should not show error when dashboard fetch request is cancelled (metabase#47170)", () => {
+    cy.visit(`/dashboard/${ORDERS_DASHBOARD_ID}`);
+
+    appBar().button("Toggle sidebar").click();
+    navigationSidebar().findByText("Dashboard A").click();
+    appBar().button("Toggle sidebar").click();
+    navigationSidebar().findByText("Orders in a dashboard").click();
```

Comment:
Right now this test visits a dashboard 3 times but 2 times are enough to reproduce the issue.